packs/data/fists-of-the-ruby-phoenix-bestiary.db/takatorra-level-13.json

```diff
@@ -728,7 +728,98 @@
                 "requirements": {
                     "value": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "domain": "all",
+                        "key": "RollOption",
+                        "option": "blade-barrage",
+                        "suboptions": [
+                            {
+                                "label": "PF2E.NPCAbility.Takatorra.BladeBarrage.StrikeZero",
+                                "value": "zero"
+                            },
+                            {
+                                "label": "PF2E.NPCAbility.Takatorra.BladeBarrage.StrikeOne",
+                                "value": "one"
+                            },
+                            {
+                                "label": "PF2E.NPCAbility.Takatorra.BladeBarrage.StrikeTwo",
+                                "value": "two"
+                            },
+                            {
+                                "label": "PF2E.NPCAbility.Takatorra.BladeBarrage.StrikeThree",
+                                "value": "three"
+                            },
+                            {
+                                "label": "PF2E.NPCAbility.Takatorra.BladeBarrage.StrikeFour",
+                                "value": "four"
+                            }
+                        ],
```

Comment:
Ok, I take it back I think this maybe should be an effect with a counter. See Marilith (I think I moved that to a counter recently).

Sorry 😅